dom.go

```diff
@@ -19,9 +19,16 @@ func Render(comp Component, container *js.Object) {
 // RenderAsBody renders the given component as the body of the page, replacing
 // whatever existing content in the page body there may be.
 func RenderAsBody(comp Component) {
-	body := js.Global.Get("document").Call("createElement", "body")
+	doc := js.Global.Get("document")
+	body := doc.Call("createElement", "body")
 	Render(comp, body)
-	js.Global.Get("document").Set("body", body)
+	if doc.Get("readyState").String() == "loading" {
```

Comment:
I think there is a zero possibility of race condition here, because JS doesn't run async with browser rendering (JS pauses browser rendering). This might not hold 100% true for gopherjs goroutines.. though I'm not sure.

For record, at least jQuery's `$.ready(...)` method is also implemented in this same fashion: https://github.com/jquery/jquery/blob/305f193aa57014dc7d8fa0739a3fefd47166cd44/src/core/ready.js#L67-L84